lib/logstash/agent.rb

```diff
@@ -175,10 +182,7 @@ def execute
   end # def execute
 
   def shutdown(pipeline)
-    pipeline.shutdown do
-      InflightEventsReporter.logger = @logger
-      InflightEventsReporter.start(pipeline.input_to_filter, pipeline.filter_to_output, pipeline.outputs)
-    end
+    pipeline.shutdown
   end
```

Comment:
Since we are removing most the content of this method, is there any reason not to call pipeline.shutdown directly and remove the inderection?